universal-application-tool-0.0.1/app/assets/stylesheets/styles.css

```diff
@@ -23,4 +23,12 @@
   button:hover {
     @apply bg-blue-700;
   }
+
+  .cf-accordion-visible .cf-accordion-button {
+    @apply rotate-180;
```

Comment:
(blocking?): https://caniuse.com/?search=%40apply seems like we shouldn't be using `@apply`.

Reply:
The CSS @apply rule is different from tailwind's @apply directive.